README.md

```diff
@@ -69,18 +88,31 @@ metadata = json.loads(r.text)
 
 # create an instance of SentinelSTAC for this scene
 item = sentinel_s2_l1c(metadata, base_url='s3://sentinel-s2-l1c/tiles/17/T/KE/2015/10/23/0')
-
+```
 
 Note however that in this example, the base_url of s3://sentinel-s2-l1c, is a requester-pays bucket. It will be used to generate the links to the assets as seen in the [Sentinel-2 L1C example](samples/sentinel-s2-l1c_item.json), but none of these files are accessed directly.
 
-However, for Sentinel-1, there is an additional metadata file that is needed that is only available in the bucket. **If you have credentials defined when running this code, it will automatically use requester-pays and you will be charged!** 
+However, for Sentinel-1, there is an additional metadata file that is needed that is only available in the bucket. **If you have credentials defined when running this code, it will automatically use requester-pays and you will be charged!**
 
 
 ## STAC conversion notes
 
 ### Sentinel-1
 
-The main metadata for a Sentinel-1 scene is contained in the `productInfo.json` file, but it does not contain all metadata needed. For each Sentinel-1 asset in the scene there is an XML annotation file (in the annotation/ subdirectory). Most of the data in the annotation file is the same across all annotation files within the scene, so only a single one is needed (there is asset specific data including statistics and noise estimates that are not needed). This annotation file is fetched, and usedto fill in the additional STAC metadata.
+Sentinel-1 RTC images are derived from multiple Sentinel-1 GRD frames. Full documentation here: https://registry.opendata.aws/sentinel-1-rtc-indigo/. We extract key metadata stored as GEOTIFF tags in `local_incident_angle.tif`. Additional extensive metadata from the GRDs are available `manifest.safe` and `productInfo.json`. An example folder structure looks like the following for `s3://sentinel-s1-rtc-indigo/tiles/RTC/1/IW/12/S/YJ/2016/S1B_20161121_12SYJ_ASC`:
+```
+.
+├── Gamma0_VH.tif
+├── Gamma0_VV.tif
+├── README.txt
+├── S1B_IW_GRDH_1SDV_20161121T010910_20161121T010939_003050_0052FC_EC22
+│   ├── manifest.safe
+│   └── productInfo.json
+├── S1B_IW_GRDH_1SDV_20161121T010939_20161121T011004_003050_0052FC_3426
+│   ├── manifest.safe
+│   └── productInfo.json
+└── local_incident_angle.tif
```

Comment:
note directory layout w/o a single JSON file that contains metadata necessary for a STAC entry